File: src/main/java/com/example/climbingapi/model/Route.java
package com.example.climbingapi.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.OffsetDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Route {

    private Integer id;
    private Integer wallId;
    private String name;
    private String grade;
    private Integer length;
    private String style;
    private Integer bolts;
    private Integer ropeLengths;
    private String firstAscendant;
    private String description;
    private OffsetDateTime createdAt;
}

File: src/main/kotlin/com/example/climbingapi/model/Route.kt
package com.example.climbingapi.model

import java.time.OffsetDateTime

data class Route(
    val id: Int?,
    val wallId: Int?,
    val name: String?,
    val grade: String?,
    val length: Int?,
    val style: String?,
    val bolts: Int?,
    val ropeLengths: Int?,
    val firstAscendant: String?,
    val description: String?,
    val createdAt: OffsetDateTime?
)
